app/webrtc/server/settings.js

```diff
@@ -1,24 +1,34 @@
 import { settings } from '../../settings';
 
 settings.addGroup('WebRTC', function() {
+	this.add('WebRTC_Enabled', false, {
```

Comment:
Couple of points here
1. I see you've defined this setting here however I don't see this setting being used anywhere except within the new file `app/webrtc/client/tabBar.tsx` which you've created in this PR. Since you're introducing this new setting here, we'll need to make sure we enforce this setting in all the relevant places in the codebase.
2. We'll be needing a DB migration for this one. I didn't see any migrations for this in your PR.